ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -405,6 +408,59 @@ S3Terminate(WT_STORAGE_SOURCE *storage, WT_SESSION *session)
     return (0);
 }
 
+/*
+ * S3Flush --
+ *     Flush file to S3 Store using AWS PutObject.
+ */
+static int
+S3Flush(WT_STORAGE_SOURCE *storageSource, WT_SESSION *session, WT_FILE_SYSTEM *fileSystem,
+  const char *source, const char *object, const char *config)
+{
+    S3_FILE_SYSTEM *fs = (S3_FILE_SYSTEM *)fileSystem;
+    int ret = fs->connection->PutObject(fs->bucketName, object, source);
+    return (ret);
+}
+
+/*
+ * S3FlushFinish --
+ *     Flush local file to cache.
+ */
+static int
+S3FlushFinish(WT_STORAGE_SOURCE *storage, WT_SESSION *session, WT_FILE_SYSTEM *fileSystem,
+  const char *source, const char *object, const char *config)
+{
+    S3_STORAGE *s3 = (S3_STORAGE *)storage;
+    std::string srcPath = S3HomePath(fileSystem, source);
+    std::string destPath = S3CachePath(fileSystem, source);
+    std::cout << srcPath << destPath << std::endl;
```

Comment:
Is this debugging code? I think you want to remove this line?

Reply:
Yes! Thank you for pointing this out.